frontend/src/lib/components/ViewRecordingButton/ViewRecordingButton.tsx

```diff
@@ -14,6 +14,17 @@ import { MatchedRecording } from '~/types'
 
```

Comment:
Not a big fan of the new layout in general.

Now in the left column we have either sparkline or issue view. Sparkline view feels squeezed while the isssue view feels like we are wasting space - it is unnecessarily wide.
| Sparkline | Issue view |
|--------|--------|
| <img width="1415" height="945" alt="image" src="https://github.com/user-attachments/assets/1ff8f374-a114-4d8d-9451-07d5ace80cbc" /> | <img width="1262" height="723" alt="image" src="https://github.com/user-attachments/assets/b5686419-5237-4197-b8e5-0a85826c254a" /> |

Reply:
I think current layout is cool with slight suggestions:
- decrease sparkline height (saves Y space),
- make stack trace show relevant frames only by default (saves Y space, sentry has it),
- now we gained a lot of Y space -  we can make events scroll within its container (we solve problem of awkward events scrolling),
- move everything issue related to the top,
- use right side of the screen for tabs and put stuff like discussions, external references, AI related things or simillar issues here

<img width="1464" height="845" alt="image" src="https://github.com/user-attachments/assets/06ff1b5d-0bb0-440c-a2c3-1d3c5d2e3b51" />